docs/contributing.md

```diff
@@ -36,6 +36,10 @@ A trusted group of core developers are responsible for maintaining the PrairieLe
 
 ### Code review guidelines
 
+- **Review lifecycle**
+  - After all comments have been addressed or responded to, the author should re-request review from the primary reviewer.
+    - This is an important step to let the reviewer know that you are done addressing their comments, and are ready for another review.
+
```

Comment:
Nit: order this at the end of this section? Reading from the top down, that'd make this sort of sequential (self review first, you'll get comments from other people, address them, request review again).

Honestly this could just be tacked onto the end of the "Addressing code review comments" section as a final bullet, for the same reason as above. This is not as much of a nit, I think we should do this.